mlos_core/mlos_core/tests/optimizers/optimizer_test.py

```diff
@@ -1,5 +1,3 @@
-#
-# Copyright (c) Microsoft Corporation.
```

Comment:
is it not copyrighted by Microsoft anymore? @bpkroth do you know what's our policy?

Reply:
Whoops that's on me. That's embarrassing I did not intend to remove those lines.